package.json

```diff
@@ -10,15 +10,15 @@
     "build": "lerna run build",
     "bootstrap": "lerna clean && lerna bootstrap",
     "test": "lerna run test",
-    "lint": "eslint --cache packages --fix",
-    "lint:fix": "npm run lint --fix",
+    "lint": "lerna run lint",
+    "lint:fix": "lerna run lint --fix",
```

Comment:
Does this work properly? I believe it should add an extra set of `--` 
`"lint:fix": "lerna run lint -- --fix",`

Reply:
oh could be

Reply:
it was: `lerna run lint -- -- --fix` like nonononohoist